news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges.md

```diff
@@ -0,0 +1,83 @@
+- **Uniqueness**: Throughout the history of osu!, we can see that beatmaps displaying unique patterns usually be the centre of community attention and sometimes even become era-defining masterpieces. With that in mind, we would like to draw players' attention to the artistic aspects of mapping and see where their opinions revolve.
+
+Additionally, we will also entertain a **Player's Impression** section. Players are asked to note down their personal thoughts and subjective opinions regarding the beatmap and rate it accordingly.
+
+To take part as a Players' Pick judge, your osu! rank needs to be *higher than #5000* as the initial requirement. We're planning to recruit **5 judges** in total for this section of the contest. So if you're a professional osu! player and like playing a variety of beatmaps, don't miss out on becoming a Players' Pick judge!
+
+### Multi-mode Judges
+
+Drummers, fruit catchers, and keyboard smashers behold! This year, we are going to host three more contests for three other game modes. The details are still worked on, but if you're an early bird wishing to partake as a judge for osu!taiko, osu!catch, or osu!mania edition of the osu! Beatmapping World Championship, you may submit your application right away!
+
+## Schedule
+
+Below is the provisional schedule for the upcoming contest. Be reminded that reschedules might occur due to unexpected organizational matters.
+
+| Event | Timestamp |
+| --: | :-- |
+| o!BWC Announcement | 2021-05-02 |
+| Registration Phase | 2021-05-02/2021-05-16 |
+| First Round - Mapping phase | 2021-05-17/2021-05-31 |
+| First Round - Judging phase | 2021-06-01/2021-06-15 |
+| Second Round - Mapping phase | 2021-06-16/2021-06-30 |
+| Second Round - Judging phase | 2021-07-01/2021-07-15 |
+| Third Round - Mapping phase | 2021-07-16/2021-07-30 |
+| Third Round - Judging phase | 2021-07-31/2021-08-14 |
+| Fourth Round - Mapping phase | 2021-08-15/2021-08-29 |
+| Fourth Round - Judging phase | 2021-08-30/2021-09-13 |
+| Results Announcement & Livestream | 2021-09-15 |
+
+Multi-mode contests are planned to be hosted approximately 2 months anterior to the conclusion of the osu! Beatmapping World Championship. We are trying our best to finalize things in that regard as quick as possible, so keep a lookout!
+
+## Applications
+
+Interested in becoming a judge? Awesome! Whichever judging section you wish to take part in, all you have to do is to click on the link below.
+
+[**Please fill out this form in order to apply as a judge!**](https://forms.gle/he4sT7DeX5UVr4j27)
+
+---
+
+Ultimately, it is time to wrap things up. We hope that 2021 edition of osu! Beatmapping Championship, with its traditional and brand-new features, has piqued your interest. For forthcoming updates, [follow us on Twitter](https://twitter.com/osubwc) and [join our Discord server](https://discord.gg/CZp4bNx)!
```

Comment:
```suggestion
Unfortunately, it is time to wrap things up. We hope that the 2021 edition of osu! Beatmapping Championship, with its traditional and brand-new features, has piqued your interest. For forthcoming updates, [follow us on Twitter](https://twitter.com/osubwc "Twitter") and [join our Discord server](https://discord.gg/CZp4bNx "Discord")!
```
assuming "ultimately" meant "in conclusion", but either way sounds a little off?

Reply:
Maybe say, "With all the exciting news, it is time to wrap up."?